pkgs/development/libraries/gdal/default.nix

```diff
@@ -234,18 +234,19 @@ stdenv.mkDerivation (finalAttrs: {
   '';
   nativeInstallCheckInputs = with python3.pkgs; [
     pytestCheckHook
+    pytest-benchmark
```

Comment:
Why adding pytest-benchmark to build inputs at the moment when we want to disable it ?

Reply:
OK, I see, building without `pytest-benchmark` is causing

```
Executing pytestCheckPhase
/build/source/build/autotest /build/source/build
Copying /nix/store/kvpxdc0027xzkfwyi145r8v139wj24mf-proj-9.4.0/share/proj/proj.db to /build/source/build/autotest/gcore/tmp/proj_db_tmpdir
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --benchmark-disable
  inifile: /build/source/build/autotest/pytest.ini
  rootdir: /build/source/build/autotest
```.

But I would still prefer something less counter-intuitive.

Reply:
The tests complained about the missing fixture, and pytest-benchmark kinda offers the best targeting mechanism to disable all benchmark tests.

I also tried modifying the `disabledTestPaths`, but with limited time I could not get it to work.

Reply:
OK, thanks. For now, I can't find a better way thow to disable them either.